table-test.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>table-tets</title>
+</head>
+<body>
+<table>
```

Comment:
The code itself, in terms of syntax is perfect.
But, the formatting of the code is wrong:

1. You should reformat all indentations within your `<table>` element (`<tr>` is a child of `<thead>` or `<tbody>` or `<tfoot>`, `<th>` is child of `<tr>` and so on...) to show that relation).
2. You should spread your elements, each to its own line (with the correct indentation).

Here's how your code should look (I took your code and reformatted it):
```
<table>
    <thead>
        <tr>
            <th>my</th>
            <th>first</th>
            <th>table</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>one</td>
            <td>two</td>
            <td>three</td>
        </tr>
        <tr>
            <td>bla</td>
            <td>bla</td>
            <td>bla</td>
        </tr>
    </tbody>
    <tfoot>
        <tr>
            <td>this</td>
            <td>is</td>
            <td>the end</td>
        </tr>
    </tfoot>
</table>
```

As you can see (I hope 😄), it's much more readable that way! It indeed takes more space, but our eyes are much more intuitive with noticing new-lines as separators between different elements.